Mathlib/Combinatorics/SimpleGraph/Connectivity/Subgraph.lean

```diff
@@ -103,12 +103,53 @@ protected lemma Connected.sup {H K : G.Subgraph}
   rintro ⟨v, (hv|hv)⟩
   · exact Reachable.map (Subgraph.inclusion (le_sup_left : H ≤ H ⊔ K)) (hH ⟨u, hu⟩ ⟨v, hv⟩)
   · exact Reachable.map (Subgraph.inclusion (le_sup_right : K ≤ H ⊔ K)) (hK ⟨u, hu'⟩ ⟨v, hv⟩)
+
 end Subgraph
 
-/-! ### Walks as subgraphs -/
+/-! ### Coercion of walks-/
 
 namespace Walk
 
+variable {H : Subgraph G} {u v w : H.verts}
+
+/-- Coerces a walk in a subgraph to the walk in the ambient graph -/
+protected def coe {u v} (p : H.coe.Walk u v) : G.Walk u.val v.val :=
```

Comment:
Is that not `p.map someGraphEmbeddingHere`?

Reply:
:man_facepalming: (It is in fact `Subgraph.hom`, so that saves a lot of code:D)